src/components/views/BlackCardSelection.js

```diff
@@ -60,6 +76,32 @@ const BlackCardSelection = () => {
 
     // fetch the blackCards from the server (it is the server's responsibility to give us 8 cards)
     useEffect(() => {
+        async function fetchBlackCard() {
+            // get blackCard
+            try {
+                const response = await api.get(`users/${userId}/games/activeGame`,
+                    {
+                        // reconfiguration might be necessary in case token is not in localStorage here
+                        headers: {
+                            "authorization": token
+                        }
+                    });
+                // set black card and time from blackcard, if no blackcard an error gets thrown
+                setBlackCard(response.data.blackCard);
+                setTime(response.data.creationDate)
+            }
+            catch (error) {
+                if(error.response.status === 404){
+                    console.error("Error 404: ", error.response.data.message)
```

Comment:
I don't think we want to print an error if we use the error for the program logic.

Reply:
done